packages/core/admin/admin/src/content-manager/components/DynamicZone/components/DynamicComponent.js

```diff
@@ -134,7 +134,7 @@ export const DynamicComponent = ({
         <Drag />
       </IconButton>
       <Menu.Root>
-        <Menu.Trigger size="S" endIcon={undefined} paddingLeft={2} paddingRight={2}>
+        <Menu.Trigger size="S" endIcon={null} paddingLeft={2} paddingRight={2}>
```

Comment:
do you think the issue is here that by default we shouldn't have an `endIcon`? Think aloud here...

Reply:
Yeah we could discuss that, but I think this prop is on the list of props to remove in v2 anyway and make it 100% composable, so not sure how valuable it is to discuss it now?